tools/sigclient/pkg/alerts/alerts.go

```diff
@@ -451,6 +388,13 @@ func verifyAlertHistory(host string, alerts []*alertutils.AlertDetails) error {
 	return nil
 }
 
+func removeTrailingSlashes(url string) string {
+	for strings.HasSuffix(url, "/") {
+		url = strings.TrimSuffix(url, "/")
+	}
+	return url
+}
```

Comment:
fyi: you can use `TrimRight` and then you don't need the loop. See https://go.dev/play/p/SwbhHJU0k2W
But I think the intent is more clear how you wrote it, so I'd suggest keeping what you have

Reply:
Ohh, I actually pushed the changes without reading your comment full after looking at the playground code. 
But I believe even `TrimRight` and `/` as the param will also be clear in the same way, especially with the function name.